src/main/webapp/script.js

```diff
@@ -799,6 +796,33 @@ function changeLocation() {
  * Methods for create-event-form and edit-event form
  * **********************************************************************/
 
+/**
+ * Ensures users cannot access the create page without being logged in.
+ */
+function createVerification() {
```

Comment:
What happens if the user types in "create-event-form.html" directly as a URL?

Reply:
Oh good catch. I'll add a check on page load instead, thanks.